src/Tecan/TecanProtocol.php

```diff
@@ -2,10 +2,13 @@
 
 namespace Mll\LiquidHandlingRobotics\Tecan;
 
+use Carbon\Carbon;
+use Composer\InstalledVersions;
```

Comment:
Requires `composer/composer` (I think) - we should add an explicit dependency to make sure it is installed.